src/StackExchange.Redis/ServerEndPoint.cs

```diff
@@ -1091,6 +1093,11 @@ private void SetConfig<T>(ref T field, T value, [CallerMemberName] string? calle
                 Multiplexer?.ReconfigureIfNeeded(EndPoint, false, caller!);
             }
         }
+        internal static string ClientInfoSanitize(string? value)
+        => string.IsNullOrWhiteSpace(value) ? String.Empty : nameSanitizer.Replace(value!.Trim(), "-");
+            if (string.IsNullOrWhiteSpace(value)) return "";
```

Comment:
Sorry my suggestion borked this method. Original lines need to be cleaned up.

Reply:
yep, have already pushed fixification